mainapp/views.py

```diff
@@ -360,6 +360,11 @@ def contributors(request):
 
 
 def request_list(request):
+    deleteitem = request.GET.get('deleteitem','')
+    if deleteitem:
+        item = Request.objects.get(id=deleteitem)
+        item.status = 'del'
+        item.save()
     filter = RequestFilter(request.GET, queryset=Request.objects.exclude(status='sup') )
```

Comment:
might need to display only **New** and **In Progress** items, something like:

`filter = RequestFilter(request.GET, queryset=Request.objects.filter(status__in=['new','pro']) )`